addons/pos_hr/models/pos_order.py

```diff
@@ -14,6 +14,12 @@ def _order_fields(self, ui_order):
         order_fields['employee_id'] = ui_order.get('employee_id')
         return order_fields
 
+    def get_table_draft_orders(self, table_id):
+        orders = super(PosOrder, self).get_table_draft_orders(table_id)
+        for order in orders:
+            order['employee_id'] = order['employee_id'][0] if order['employee_id'] else False
```

Comment:
remove the function "get_table_draft_orders", because it calls a domain with "table_id" in point_of_sale which does not exist.

And so in pos restaurant after the search read do something like:
for order in orders:
  if order('employee_id', False):
    order['employee_id'] = order['employee_id'][0] if order['employee_id'] else False